apps/web/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": { "types": ["vite/client"] },
+  "extends": "@umami/typescript-config/tsconfig.json",
+  "include": ["src", "vite.config.ts", "jest.config.js"],
```

Comment:
I think we should stick to jest.config.ts instead, otherwise, as I understand, this inclusion is not helpful at all